plugins/mcreator-localization/lang/texts.properties

```
@@ -1216,6 +1217,7 @@ modelement.food.description=Food is an item that can be eaten and can give you p
 modelement.fuel.description=Fuel is a block or item that can be used as fuel in furnaces.
 modelement.function.description=Function allows players to run command sets when called.
 modelement.gui.description=GUI is a displayable panel with elements such as buttons, labels, slots, ...
+modelement.itemextension.description=An Item extension is an extra property that can be added to an item.
```

Other mod elements don't have articles (a/an/the) in their descriptions.
```suggestion
modelement.itemextension.description=Item extension is a set of extra properties that can be added to an item.
```